pkg/roachprod/roachprod.go

```diff
@@ -67,6 +67,11 @@ const (
 	prometheusHostUrlEnv = "COCKROACH_PROM_HOST_URL"
 )
 
+// supportedPromProjects are the projects supported for prometheus target
+var supportedPromProjects = map[string]struct{}{
+	"cockroach-ephemeral": {},
```

Comment:
If you are planning to use `GC` then you can probably leverage `DefaultProject()` from `gcloug.go` to keep this in sync.